src/dataviews/histogram-dataview-model.js

```diff
@@ -208,7 +208,9 @@ module.exports = DataviewModelBase.extend({
   _onColumnChanged: function () {
     this._totals.set({
       column_type: this.get('column_type'),
-      column: this.get('column')
+      column: this.get('column'),
+      start: null,
```

Comment:
You didn't take into account the public API while doing these changes. You are going to mislead CARTO.js developers. 😂

We need to override `.setColumn` to reset start and end values in carto.dataviews.Histogram, because if we don't do that we are going to return the wrong values when invoking `.getStart` and `.getEnd`.

Reply:
Good catch! 👀

Reply:
I don't want developers to say `pero que ha pachao` 😂

Reply:
The problem is we don't have any way to know if the changes are because of the public API or the internal one.

If we reset `start` and `end`, we break the workflow in the public API, because users won't expect the start and end to change, if we don't reset it we break it in builder...

I think it makes sense the `start` and `end` values are reset since probably they won't have that much sense with the new column, maybe we can just document it in the public API?

Reply:
Yes, we do have a way 🤔 Do you want me to tell you about it, or you want to discover it by yourself? 🕵️